include/boost/decimal/decimal32.hpp

```diff
@@ -2368,7 +2377,9 @@ constexpr auto floord32(decimal32 val) noexcept -> decimal32
     }
 
     auto new_sig {val.full_significand()};
-    auto abs_exp {std::abs(val.biased_exponent())};
+
+    auto abs_exp = val.biased_exponent(); if (abs_exp < 0) { abs_exp = -abs_exp; }
+
```

Comment:
This feels like a more verbose way to the same end.

Reply:
Indeed. But I had a compiler that failed to have the `constexpr` version of `<algorithm>`'s function `std::abs()`. It was MSVC 142. So I kind of _had_ _to_ do this. SIgh.

Reply:
I dream of a day where MSVC works like everyone else.